ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/process/search_tree/Tree.java

```diff
@@ -86,6 +102,33 @@ private Tree() {
         return CompletableFuture.completedFuture(buildOutput(rootLeaf, optimalLeaf));
     }
 
+    /**
+     * Stop criterion check: maximum research depth reached
+     */
+    static boolean goDeeper(int currentDepth, int maxDepth) {
```

Comment:
private

Reply:
Can I put it protected, in order to keep the test I wrote in TreeTest.java?